src/current/v23.1/essential-metrics.md

```diff
@@ -0,0 +1,135 @@
+| storage.write-stalls                                | storage.write.stalls                                         | Storage                 | Number of instances of intentional write stalls to backpressure incoming writes | This metric reports actual disk stall events. Ideally, investigate all reports of disk stalls. As a pratical guideline, one stall per minute is not likely to have a material impact on workload beyond an occasional increase in response time. However one stall per second should be viewed as problematic and investigated actively.  It is particularly problematic if the rate persists over an extended period of time, and worse, if it is increasing. |
+| rocksdb.compactions                                 | rocksdb.compactions.total                                    | Storage                 | Number of SST compactions                                    | This metric reports the number of a node's [LSM compactions](common-issues-to-monitor.html#lsm-health). If the number of compactions remains elevated while the LSM health does not improve, compactions are not keeping up with the workload. If the condition persists for an extended period, the cluster will initially exhibit performance issues that will eventually escalate into stability issues. |
+| rocksdb.block.cache.hits                            | rocksdb.block.cache.hits                                     | Storage                 | Count of block cache hits                                    | This metric gives hits to block cache which is reserved memory. It is allocated upon the start of a node process by the [`--cache` flag](cockroach-start.html#general) and never shrinks. By observing block cache hits and misses, you can fine-tune memory allocations in the node process for the demands of the workload. |
+| rocksdb.block.cache.misses                          | rocksdb.block.cache.misses                                   | Storage                 | Count of block cache misses                                  | This metric gives misses to block cache which is reserved memory. It is allocated upon the start of a node process by the [`--cache` flag](cockroach-start.html#general) and never shrinks. By observing block cache hits and misses, you can fine-tune memory allocations in the node process for the demands of the workload. |
+| rocksdb.read-amplification                          | rocksdb.read.amplification                                   | Storage                 | Number of disk reads per query                               | This metric measures the worst-case or max number of files to read to serve a single key. It measures the [health of an LSM tree](common-issues-to-monitor.html#lsm-health). |
+| sys.uptime                                          | sys.uptime                                                   | Health                  | Process uptime                                               |                                                              |
```

Comment:
We're missing sys.uptime usage description.

Reply:
Potential usage description: The sys.uptime metric measures the length of time, in seconds, that the CockroachDB process has been running. Monitoring the sys.uptime metric can help you detect events such as node restarts, which may require investigation or intervention.

Reply:
added usage for `sys.uptime`